packages/webpack-extraction-plugin/src/GriffelCSSExtractionPlugin.ts

```diff
@@ -77,8 +81,14 @@ function ensureModuleHasPostOrderIndex(griffelChunk: Chunk, cssModule: Module) {
   }
 }
 
-function moveCSSModulesToGriffelChunk(compilation: Compilation, chunks: Iterable<Chunk>, griffelChunk: Chunk) {
-  for (const chunk of chunks) {
+function moveCSSModulesToGriffelChunk(compilation: Compilation) {
+  const griffelChunk = compilation.namedChunks.get('griffel');
+
+  if (!griffelChunk) {
+    return;
```

Comment:
if there is no griffel chunk should we warn?

Reply:
No, Webpack can run in child compiler mode and it's expected to not have a griffel chunk in this case.